specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/Sessions.json

```diff
@@ -0,0 +1,387 @@
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/SessionPoolNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/SessionCollection"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List Sessions by Session Pool": {
+            "$ref": "./examples/Sessions_ListBySessionPool.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "SSHConfiguration": {
+      "description": "SSH configuration of the Session.",
+      "type": "object",
+      "properties": {
+        "username": {
+          "description": "The username.",
+          "type": "string"
+        },
+        "password": {
+          "description": "The password.",
+          "type": "string"
+        },
+        "authorizedPublicKey": {
+          "description": "The authorized public key.",
+          "type": "string"
+        }
+      }
+    },
+    "CodeExecutionConfiguration": {
+      "description": "Code execution configuration of the Session.",
+      "type": "object",
+      "properties": {
+        "token": {
+          "description": "The token used for code execution.",
+          "type": "string"
+        },
+        "environment": {
+          "description": "The environment used for code execution.",
+          "type": "string"
+        }
+      }
+    },
+    "SessionTemplate": {
+      "description": "Session template.",
+      "type": "object",
+      "properties": {
+        "requestedDurationInSeconds": {
+          "description": "The requested duration of the Session in seconds.",
+          "type": "integer",
+          "format": "int32"
+        },
+        "timeElapsedInSeconds": {
+          "description": "The time elapsed in seconds since the Session was created.",
+          "type": "integer",
+          "format": "int32",
+          "readOnly": true
+        },
+        "expiredAt": {
+          "description": "The time when the Session will expire.",
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true
+        },
+        "generatedAt": {
+          "description": "The time when the Session was generated.",
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true
+        },
```

Comment:
mention the format/timezone of time in description

Reply:
Fixed

Reply:
Description say it is timestamp but data type is `date-time` not integer. If is ISO 8601 date format, then mentioned ISO 8601 date time in UTC format.